extension/test/data_test.ts

```diff
@@ -69,4 +69,166 @@ describe('data.ts', function () {
       ).to.include.something.like({ entry: /^ＤＶＤ-ＲＯＭ .*/ });
     });
   });
+
+  describe('makeText', function () {
+    describe('with a word dict entry', function () {
+      // wordDictEntry was generated with: rcxDict.wordSearch('あ', /* doNames= */ false)
+      const wordDictEntry = {
+        kanji: '',
+        onkun: '',
+        nanori: '',
+        bushumei: '',
+        misc: {},
+        eigo: '',
+        hasNames: false,
+        data: [
+          { entry: 'あ /(int) (1) ah/oh/(int) (2) hey!/', reason: undefined },
+          { entry: 'ア /(int) (1) ah/oh/(int) (2) hey!/', reason: undefined },
+        ],
+        hasMore: false,
+        title: '',
+        index: 0,
+        matchLen: 1,
+      };
+
+      it('when max is 1, returns 1 data entry as text', function () {
+        const singleDataEntryAsText = rcxDict.makeText(
+          wordDictEntry,
+          /* max= */ 1
+        );
+
+        expect(singleDataEntryAsText).to.equal(
+          'あ\t(int) (1) ah; oh; (int) (2) hey!\n'
+        );
+      });
+
+      it('when max is 2, returns 2 data entries as text', function () {
+        const twoDataEntriesAsText = rcxDict.makeText(
+          wordDictEntry,
+          /* max= */ 2
+        );
+
+        expect(twoDataEntriesAsText).to.equal(
+          'あ\t(int) (1) ah; oh; (int) (2) hey!\nア\t(int) (1) ah; oh; (int) (2) hey!\n'
+        );
+      });
+
+      it('when max is greater than the number of data entries, returns all data entries as text', function () {
+        const allDataEntriesAsText = rcxDict.makeText(
+          wordDictEntry,
+          /* max= */ 1000
+        );
+
+        expect(allDataEntriesAsText).to.equal(
+          'あ\t(int) (1) ah; oh; (int) (2) hey!\nア\t(int) (1) ah; oh; (int) (2) hey!\n'
+        );
+      });
+    });
+
+    describe('with a name dict entry', function () {
+      // nameDictEntry was generated with: rcxDict.wordSearch('あ', /* doNames= */ true)
+      const nameDictEntry = {
+        kanji: '',
+        onkun: '',
+        nanori: '',
+        bushumei: '',
+        misc: {},
+        eigo: '',
+        hasNames: true,
+        data: [
+          { entry: '亜 [あ] /(f) A/', reason: undefined },
+          { entry: '阿 [あ] /(s) A/', reason: undefined },
+        ],
+        hasMore: false,
+        title: '',
+        index: 0,
+        matchLen: 1,
+      };
+
+      it('when max is 1, returns 1 data entry as text', function () {
```

Comment:
Relevent to the comment about needing these tests; even if we thought a name test was useful, we probably don't need to test as many cases as before. A single test focusing on the fact the that it's a name is enough given the behavior of `max` isn't different at all. (This won't matter if we delete all of them of course)

Reply:
I ended up deleting these tests because `makeText` doesn't treat name dict entries any differently than word dict entries.